analysis/lang/cro/analyzer_cro.go

```diff
@@ -0,0 +1,64 @@
+//  Copyright (c) 2018 Couchbase, Inc.
```

Comment:
you may update the copyright dates here?

Reply:
Sure! :)